doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -11,11 +11,11 @@ This tutorial has following sections:
 - [Azure SDK Code Generation Quickstart Tutorial (Data Plane)](#azure-sdk-code-generation-quickstart-tutorial-data-plane)
   - [Prerequisites](#prerequisites)
```

Comment:
Please make the pre-requisites its own document and simply have one sentence here along the lines of "For first time setup of a new package verify you have met the prerequisites."

Reply:
updated

Reply:
We still have the `Create SDK Package` which is already in the `Prerequisites` link.  Can we remove this.

Reply:
updated.